service/mcs/driver/src/main/java/eu/europeana/cloud/mcs/driver/DataSetServiceClient.java

```diff
@@ -502,21 +503,22 @@ public void unassignRepresentationFromDataSet(String providerId, String dataSetI
     }
 
     /**
-     * Retrieve chunk of cloudIds from data set for specific revision.
+     * Retrieve chunk of cloudIds and tags from data set for specific revision.
      *
      * @param providerId         provider identifier (required)
      * @param dataSetId          data set identifier (requred)
      * @param representationName name of the representation (required)
      * @param revisionName       revision naem (required)
      * @param revisionProviderId revision provider id (required)
+     * @param revisionTimestamp  timestamp of the searched revision which is part of the revision identifier
      * @param startFrom          code pointing to the requested result slice (if equal to
      *                           null, first slice is returned)
-     * @return chunk of representation cloud identifier list from data set
+     * @return chunk of representation cloud identifier list from data set together with tags of the revision
      * @throws MCSException on unexpected situations
      */
-    public ResultSlice<String> getDataSetRevisionsChunk(String providerId, String dataSetId,
-                                                        String representationName, String revisionName,
-                                                        String revisionProviderId, String startFrom)
+    public ResultSlice<CloudTagsResponse> getDataSetRevisionsChunk(String providerId, String dataSetId,
+                                                                   String representationName, String revisionName,
```

Comment:
No limitParam is added to the signeture of those clients making those clients unable to request the rest api ! .